docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -1880,31 +1426,31 @@ comment: |
   This block is called when the "field table" is initially loaded.
 code: |
   # Initialize the hidden fields. 
-  questions[i].fld_order_list = '' # Will be updated in js
-  questions[i].table_data = ''     # Will be updated in the next code block
+  interview.questions[i].fld_order_list = '' # Will be updated in js
+  interview.questions[i].table_data = ''     # Will be updated in the next code block
   
   # Make the field table draggable  
-  questions[i].html_fld_table = make_it_draggable(questions[i].edit_table) 
-  questions[i].original_fld_order = questions[i].html_fld_table[0] #list
-  questions[i].draggable_fld_table = questions[i].html_fld_table[1] #soup table
+  interview.questions[i].html_fld_table = make_it_draggable(interview.questions[i].edit_table) 
+  interview.questions[i].original_fld_order = interview.questions[i].html_fld_table[0] #list
+  interview.questions[i].draggable_fld_table = interview.questions[i].html_fld_table[1] #soup table
 ---
 comment: |
   This block is called after the "field table" is reordered by the user.
 code: |     
-  if len(questions[i].fld_order_list) > 0:    
+  if len(interview.questions[i].fld_order_list) > 0:    
     # Update existing fld table order var - will affect output interview layout
-    questions[i].field_list = update_table_order_var(questions[i].original_fld_order, questions[i].fld_order_list, questions[i].field_list)
+    interview.questions[i].field_list = update_table_order_var(interview.questions[i].original_fld_order, interview.questions[i].fld_order_list, interview.questions[i].field_list)
     
     # Save new table data to the hidden field for JS to access    
-    questions[i].table_data = make_json(questions[i].fld_order_list, questions[i].draggable_fld_table)    
+    interview.questions[i].table_data = make_json(interview.questions[i].fld_order_list, interview.questions[i].draggable_fld_table)    
     
-  questions[i].field_tbl_done = True  
+  interview.questions[i].field_tbl_done = True  
 ---
 # Loop thru all the field tables and display them on the screen
 code: |  
-  for j in range(0, len(questions)):      
-    questions[j].edit_question     
-    questions[j].field_tbl_done 
+  for index in range(0, len(interview.questions)):      
+    interview.questions[index].edit_question     
+    interview.questions[index].field_tbl_done 
```

Comment:
```suggestion
  for question in interview.questions:      
    questions.edit_question     
    questions.field_tbl_done 
```
This should work, right?

Reply:
I think it needs to be complete_elements() if we do it that way

Reply:
gotcha, thought there might be some weird DA-ness there. Probably not worth the change and testing for the slight bump in clarity